src/msw/listctrl.cpp

```diff
@@ -334,6 +334,11 @@ void wxListCtrl::MSWSetExListStyles()
         // it seems better to enable it by default than disable
         LVS_EX_HEADERDRAGDROP
     );
+
+    // If using the default background style with double buffering enabled then switch to system
+    // Using wxBG_STYLE_ERASE while will cause flicker despite double buffering
```

Comment:
I'm not sure what I meant there exactly. Apparently I got distracted while typing it out.

Anyway, are you suggesting just adding a default wxEVT_ERASE_BACKGROUND handler such as:
`
// If double buffered do nothing in the paint event, otherwise skip it
event.Skip(!IsDoubleBuffered());
`